kcidb/db/argparse.py

```diff
@@ -0,0 +1,211 @@
+    )
+
+
+class ArgumentParser(kcidb.argparse.ArgumentParser):
+    """
+    Command-line argument parser with common database arguments added.
+    """
+
+    def __init__(self, driver_types, *args, database=None, **kwargs):
+        """
+        Initialize the parser, adding common database arguments.
+
+        Args:
+            driver_types: A dictionary of known driver names and types
+            args:       Positional arguments to initialize ArgumentParser
+                        with.
+            database:   The default database specification to use, or None to
+                        make database specification required.
+            kwargs:     Keyword arguments to initialize ArgumentParser with.
+        """
+        assert isinstance(driver_types, dict)
+        assert all(isinstance(key, str) for key in driver_types.keys())
+        assert all(isinstance(value, type) for value in driver_types.values())
+        self.driver_types = driver_types
+        super().__init__(*args, **kwargs)
+        add_args(self, database=database)
+
+
+class OutputArgumentParser(kcidb.argparse.OutputArgumentParser):
+    """
+    Command-line argument parser for tools outputting JSON,
+    with common database arguments added.
+    """
+
+    def __init__(self, driver_types, *args, database=None, **kwargs):
+        """
+        Initialize the parser, adding JSON output arguments.
+
+        Args:
+            driver_types: A dictionary of known driver names and types
+            args:       Positional arguments to initialize ArgumentParser
+                        with.
+            database:   The default database specification to use, or None to
+                        make database specification required.
+            kwargs:     Keyword arguments to initialize ArgumentParser with.
+        """
+        assert isinstance(driver_types, dict)
+        assert all(isinstance(key, str) for key in driver_types.keys())
+        assert all(isinstance(value, type) for value in driver_types.values())
+        self.driver_types = driver_types
+        super().__init__(*args, **kwargs)
+        add_args(self, database=database)
+
+
+class SplitOutputArgumentParser(kcidb.argparse.SplitOutputArgumentParser):
+    """
+    Command-line argument parser for tools outputting split-report streams,
+    with common database arguments added.
+    """
+
+    def __init__(self, driver_types, *args, database=None, **kwargs):
+        """
+        Initialize the parser, adding split-report output arguments.
+
+        Args:
+            driver_types: A dictionary of known driver names and types
+            args:       Positional arguments to initialize ArgumentParser
+                        with.
+            database:   The default database specification to use, or None to
+                        make database specification required.
+            kwargs:     Keyword arguments to initialize ArgumentParser with.
```

Comment:
Could you align those descriptions on the same column, same as was before your change, but now further to the right? Re-wrap them as necessary to match the column limit, please.